### PR TITLE
fix Issue 15133 - Error message is incomprehensible

### PR DESCRIPTION
**Pull request by CyberShadow**
https://issues.dlang.org/show_bug.cgi?id=15133

### CODE REVIEW

File: std/algorithm/iteration.d [hunk 1]
```diff
@@ -2478,7 +2478,7 @@ template reduce(fun...) if (fun.length >= 1)
 
         static if (isInputRange!R)
         {
-            enforce(!r.empty);
+            enforce(!r.empty, "Cannot reduce an empty input range w/o an explicit seed value.");
```
Comment (quickfur) [inline std/algorithm/iteration.d]: What's the reason for writing `w/o` instead of spelling out `without`?

Comment (CyberShadow) [inline std/algorithm/iteration.d]: I'm just copying the similar error message that's a bit lower for iterables.

Comment (quickfur) [inline std/algorithm/iteration.d]: Fair enough.

**Comment by quickfur** [comment]
Auto-merge toggled on

**Pull request merged**